stack-9.0.2.yaml

```diff
@@ -1,21 +1,10 @@
-resolver: nightly-2022-03-10
+resolver: lts-19.0
 compiler: ghc-9.0.2
 compiler-check: match-exact
 
-# extra-deps:
-# # Andreas, 2021-11-13
-# # happy-1.21.0 is deprecated, but currently in nightly
-# # (commercialhaskell/stackage#6294)
-# - happy-1.20.0
-
 flags:
   transformers-compat:
     five-three: true
```

Comment:
This flag is now also obsolete, see https://github.com/commercialhaskell/stackage-snapshots/blob/e5c69335723d8c92f607b672bf177c3d1b6ccbf1/lts/19/0.yaml#L44-L45